src/kademlia/node.cpp

```diff
@@ -436,12 +435,12 @@ void node::get_item(sha1_hash const& target
 #endif
 
 	boost::intrusive_ptr<dht::get_item> ta;
-	ta.reset(new dht::get_item(*this, target, f));
+	ta.reset(new dht::get_item(*this, target, boost::bind(f, _1), find_data::nodes_callback()));
```

Comment:
ah, I see. I thought I had transitioned everything to shared_ptr (and I missed that you said intrusive_ptr right above). never mind then